### ISSUE TITLE
Visual Studio 2019 16.8.0 update (dotnet 5 SDK) breaks Cake build

### ISSUE THREAD

**Issue by nf313743** [cake-build/cake#2948]
There's an issue running Cake after updating to Visual Studio 2019 to 16.8.0/16.8.1.  This update will install the new dotnet 5 SDK.  Building my solution using the Cake .netframework Runnner (build.ps1), I was receiving errors such as:
```C:\Program Files\dotnet\sdk\5.0.100\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(241,5): error NETSDK1005: Assets file 'xxx.assets.json' doesn't have a target for 'netstandard2.0'. Ensure that restore has run and that you have included 'netstandard2.0' in the TargetFrameworks for your project.```

The problem seems to be with nuget.exe. The work around to this is to delete nuget.exe in the Cake tools directory, and change the nuget URL in build.ps1 from 
```$NUGET_URL = "https://dist.nuget.org/win-x86-commandline/latest/nuget.exe"```
to
```$NUGET_URL = "https://dist.nuget.org/win-x86-commandline/v5.8.0/nuget.exe"```

Note that the 'latest' link is currently pointing to v5.7.0.  I guess this issue will sort itself out when v5.8.0 becomes the latest, but untill here's a workaround.

Thanks

**Comment by devlead** [comment]
Yes, there seems to be a breaking change in NuGet 5.8.0, for bootstrappers that don't use NuGet.exe you can add something like
```cake
#tool "nuget:https://dotnet.myget.org/F/nuget-build/api/v3/index.json?package=nuget.commandline&version=5.8.0-rc.6961&prerelease=true"
```

**Comment by pascalberger** [comment]
For bootstrapper the above workaround seems like the best solution to me until Microsoft fixes their stuff and blesses 5.8.0 to make it available as the latest version. The underlying issue is that they shipped NuGet 5.8 with Visual Studio, while not making it also the recommended version for the standalone installer, which leads to this issue due to the breaking change they introduced.

**Comment by nf313743** [comment]
`https://dist.nuget.org/win-x86-commandline/latest/nuget.exe` Now points to v5.8.  
Closing.
